### ISSUE TITLE
MySQL analyzes all tables user has access to, when specific schema is specified

### ISSUE THREAD

**Issue by hauntingEcho** [schemaspy/schemaspy#299]
When attempting to use schemaspy with a user who has access to multiple schemas, and passing a specific schema to analyze, all accessible schemas will be analyzed.  The minimal DDL I've prepared also seems to cause a `test_table_1` to be generated and displayed

## Expected Behavior
- only the specified schema should be analyzed
- `test_table_1` table should not be shown, as it does not exist

## Current Behavior
- contents of all accessible schemas are analyzed

- with the given DDL & command line, test2.a is described
- if test1 and test2 are swapped in the DDL, but the commandline is left alone, and additional table named `test_table_1` is included with columns id & name.
- if test1 & test2 are swapped in both the DDL & commandline, a table with the column `id2` (as in the view) will be listed for `a`, but no `create view` statement shown.

## Context
* Database type (e.g. MySQL): MySQL installed via homebrew.  `mysql --version` returns `mysql  Ver 14.14 Distrib 5.7.21, for osx10.13 (x86_64) using  EditLine wrapper`
* Database driver version (is you use a special one): see Docker image
* Your DDL (sample how to reproduce the issue):
```
create database test1;
create database test2;
create table test1.a(id1 int);
create view test2.a as select id1 as id2 from test1.a;
create user 'testUser'@'localhost' identified by 'password';
grant select, show view on test1.* to 'testUser'@'localhost';
grant select, show view on test2.* to 'testUser'@'localhost';
flush privileges
```

* Full command including arguments how you run SchemaSpy: `docker run -v /Users/matt.beary/Desktop/example:/output schemaspy/schemaspy:snapshot -db test1 -s test1 -u testUser -p password -t mysql -host docker.for.mac.localhost -port 3306`

## Your Environment
* Version used: see Docker image, seems to be last updated March 26
* Environment name and version (e.g. Chrome 39): `snapshot` docker image
* Operating System and version (e.g. Windows 10): OSX 10.13.4, Docker 18.03.0-ce-mac60 (23751)
* Java version: see Docker image

**Comment by npetzall** [comment]
Duplicate of #222

**Comment by npetzall** [comment]
This is really old, but #222 has been fixed and if duplicate is true this issue should now be resolved.
@hauntingEcho Can you verify that with a latest build this is resolved.

I think the problem was that a schema with no tables is deemed empty and it tries to use DatabaseMetaData.getTables and since it ignores Schema it will return the table from the other schema. So should still be solved by #222

**Comment by hauntingEcho** [comment]
this is working as expected now, when built from the master branch - thanks so much!
